packaging/Dockerfile

```diff
@@ -13,8 +13,9 @@ RUN nix-build packaging/shell.nix
 COPY . /app/
 
 RUN nix-shell packaging/shell.nix --command ./packaging/install_dependencies.sh
+RUN nix-shell packaging/shell.nix --command ./packaging/setup_backend.sh
 
 EXPOSE 5173
 EXPOSE 8000
 
-ENTRYPOINT [ "nix-shell", "packaging/shell.nix", "--command", "/app/packaging/dev.sh" ]
+ENTRYPOINT [ "nix-shell", "packaging/shell.nix", "--command", "overmind start -f packaging/Procfile" ]
```

Comment:
I'm not sure about that, as it might break bind-mounting the current source into `/app` in the container (as we describe in the dev setup docs), for example if the current source contains db migrations or new dependencies